lib/archethic/account/mem_tables_loader.ex

```diff
@@ -130,6 +130,9 @@ defmodule Archethic.Account.MemTablesLoader do
 
       {{to, _token_address}, utxo} ->
         TokenLedger.add_unspent_output(to, utxo, timestamp)
+
+      {{to, _token_address, _token_id}, utxo} ->
```

Comment:
I think the best would be to add the token_id to 0 by default so you would avoid to have 2 pattern matching.

Reply:
I think it shouldn't cause issue, as we are checking for 2 values and 3 values in struct or map. any example code to reproduce conflict.

Reply:
You have to change is also in the aggregate function

Reply:
I think it already have that pattern match,
https://github.com/prix-uniris/archethic-node/blob/3b0ad7f106fac54709076de3676f34808f0a4b7b/lib/archethic/account/mem_tables_loader.ex#L204
Where do i have to modify?

Reply:
There I think https://github.com/prix-uniris/archethic-node/blob/3b0ad7f106fac54709076de3676f34808f0a4b7b/lib/archethic/account/mem_tables_loader.ex#L223

Reply:
Alright add 0 as part of key for the map, like this
 L#223 {to, token_address, 0},
 Like this

Reply:
Or may be you can drop it, we have an another pr 
https://github.com/archethic-foundation/archethic-node/pull/557
, with test cases, which addresses this issue , more clearly.

Reply:
While Prince is on it, it will be better to handle it and clsoe the #557